lib/matplotlib/lines.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, xdata, ydata,
         Create a :class:`~matplotlib.lines.Line2D` instance with *x*
         and *y* data in sequences *xdata*, *ydata*.
 
-        The kwargs are :class:`~matplotlib.lines.Line2D` properties:
+        The *kwargs* are :class:`~matplotlib.lines.Line2D` properties:
```

Comment:
It's best to go explicit here:

~~~
Additional keyword arguments are `.Line2D` properties:
~~~